_data/companies/adp.yml

```diff
@@ -0,0 +1,6 @@
+name: ADP
```

Comment:
Is this the document/payroll processing company or someone else entirely?

Reply:
Yes, the HR/payroll company.

Reply:
Cool, thanks!

Reply:
Notifications have been sent internally -- I'm not able to provide a link to external policy page.